source/Calamari.Shared/Integration/FileSystem/CalamariPhysicalFileSystem.cs

```diff
@@ -42,7 +42,7 @@ public static CalamariPhysicalFileSystem GetPhysicalFileSystem()
         protected static RetryTracker GetRetryTracker()
         {
             return new RetryTracker(maxRetries:10000, 
```

Comment:
Consider naming to make the consuming code self-documenting: `GetDefaultRetryTracker()`